opentripplanner/otp-react-redux
# Display transit leg real-time status

This PR adds a component that displays the real-time status of a transit leg.

## To test

To test that the slot is being applied:

- Comment out the if condition at https://github.com/opentripplanner/otp-react-redux/blob/b1570b25361497e0845510cb396463a563cfc0ee/lib/components/narrative/line-itin/time-column-with-delays.js#L60
- Run `yarn start` and see the status being shown for transit legs.

To test real-time updates:

- in `trimet-mod-otp` branch master, insert in the `package.json` dependencies:
 ```json
 "@opentripplanner/itinerary-body": "^1.1.0",
```
- in `trimet-mod-otp/node_modules/otp-react-redux/node_modules`, delete the `itinerary-body` folder.
- From the configurations repo, execute `otp/run.js commtrans`. Find an itinerary with real-time updates, and observe the updated leg times per the screenshot below.

![image](https://user-images.githubusercontent.com/56846598/91188698-99b2df00-e6bf-11ea-80c5-44415a0a8dcc.png)
